tests/integration/test_integration_all_rust.py

```diff
@@ -536,22 +524,28 @@ def setup_endpoint_server():
     EP_QUEUES.extend([out_q, err_q])
 
 
-def setup_module():
-    """Set up module including BigTable or Dynamo
-    and connection, endpoint and megaphone servers.
+@pytest.fixture(autouse=True, scope="session")
+def setup_teardown() -> Generator:
+    """Set up and tear down test resources within module.
+
+    Setup: BigTable or Dynamo and connection, endpoint, mock and megaphone servers.
+
+    Using pytest fixtures for setup/teardown means anything prior to `yield` statement
+    executes at test creation and anything after `yield` is run after tests complete.
```

Comment:
nit: I think this comment repeats what is well documented by pytest and can be removed